paddle/framework/framework.proto

```diff
@@ -40,8 +40,8 @@ message OpDesc {
   };
 
   message Var {
-    required string name;                 // e.g. "X"
-    optional int dup = 2 [ default = 0 ]; // e.g., "1"
+    required string op_proto_name = 1;
```

Comment:
Agreed -- they are like parameters and argument. How about we name them in this way?